enterprise/server/workflow/service/service.go

```diff
@@ -685,15 +681,22 @@ func (ws *workflowService) InvalidateAllSnapshotsForRepo(ctx context.Context, re
 	return err
 }
 
-func (ws *workflowService) enableExtraKytheIndexingAction(ctx context.Context) (bool, error) {
+func (ws *workflowService) addDefaultConfigActions(ctx context.Context, c *config.BuildBuddyConfig, workflow *tables.Workflow, wd *interfaces.WebhookData) error {
```

Comment:
uber nit: maybe `addKytheActionIfEnabled` since "DefaultConfigActions" feels a bit ambiguous (could refer to the default "Test" action that we create when bb.yaml doesn't exist)